src/python/pants/backend/python/lint/pylint/subsystem.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.backend.python.subsystems.python_tool_base import PythonToolBase
+from pants.option.custom_types import file_option, shell_str
+
+
+class Pylint(PythonToolBase):
+  options_scope = 'pylint'
+  default_version = 'pylint'
+  default_entry_point = 'pylint'
+  default_interpreter_constraints = ["CPython>=2.7,<3", "CPython>=3.4"]
```

Comment:
Where do we get these constraints from?

Reply:
This has been updated to only allow CPython>=3.5

Source: https://github.com/PyCQA/pylint/blob/b2b19972bbd6c678d0a98e4c90c22dda602525b9/setup.py#L149